draft-ietf-quic-transport.md

```diff
@@ -2705,7 +2705,7 @@ Once the server has completed its path validation and has received a non-probing
 packet with a new largest packet number on its preferred address, the server
 begins sending non-probing packets to the client exclusively from its preferred
 IP address.  It SHOULD drop packets for this connection received on the old IP
-address, but MAY continue to process delayed packets.
+address, but can continue to process delayed packets.
```

Comment:
This is badly written -- the SHOULD and the MAY apply to different packets. How about:
"The server SHOULD drop newer packets for this connection that are received on the old IP address. The server MAY continue to process older delayed packets that are received on the old IP address."

Reply:
Please make a separate issue and pull request for this.  I agree that this is a better formulation.